README.md

```diff
@@ -129,10 +129,10 @@ mockedHTTPClient := mock.NewMockedHTTPClient(
     mock.WithRequestMatchHandler(
         mock.GetUsersByUsername,
         http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-            WriteError(
+            http.Error
```

Comment:
Hi @MichaelSp , the typo here might actually be the `WriteError` is missing the package name, it should be `mock.WriteError` like the rest of the other functions. Could you check if adding the package name makes it so it works for you?

Reply:
```suggestion
            mock.WriteError
```

yes that actually works